src/index.ts

```diff
@@ -60,4 +60,5 @@ export {
   RENDERMODE_PATHTRACE,
   RENDERMODE_RAYMARCH,
   SKY_LIGHT,
+  CameraTransform,
```

Comment:
@frasercl will this cause any issues if this type export doesn't have the `type` annotation in front of it? I get a babel warning if I include it, regardless of whether `CameraTransform` is an interface or type.